test/gpdb/scripts/test_pitr_gpdb6.sh

```diff
@@ -0,0 +1,207 @@
+MIRROR3=${DATADIR}/dbfast_mirror3/demoDataDir2
+MASTER_PORT=6000
+PRIMARY1_PORT=6002
+PRIMARY2_PORT=6003
+PRIMARY3_PORT=6004
+
+echo "Filling the pgbackrest.conf configuration file..." > /dev/null
+cat <<EOF > $PGBACKREST_TEST_DIR/pgbackrest.conf
+[seg-1]
+pg1-path=$MASTER
+pg1-port=$MASTER_PORT
+
+[seg0]
+pg1-path=$PRIMARY1
+pg1-port=$PRIMARY1_PORT
+
+[seg1]
+pg1-path=$PRIMARY2
+pg1-port=$PRIMARY2_PORT
+
+[seg2]
+pg1-path=$PRIMARY3
+pg1-port=$PRIMARY3_PORT
+
+[global]
+repo1-path=$PGBACKREST_TEST_DIR/$TEST_NAME
+log-path=$PGBACKREST_TEST_DIR/logs/$TEST_NAME
+start-fast=y
+fork=GPDB
+EOF
+
+echo "Initializing pgbackrest for GPDB..." > /dev/null
+for i in -1 0 1 2
+do 
+    PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest \
+    --config $PGBACKREST_TEST_DIR/pgbackrest.conf --stanza=seg$i stanza-create
+done
+
+echo "Configuring WAL archiving command..." > /dev/null
+gpconfig -c archive_mode -v on
+
+gpconfig -c archive_command -v "'PGOPTIONS=\"-c gp_session_role=utility\" \
+$PGBACKREST_BIN/pgbackrest --config $PGBACKREST_TEST_DIR/pgbackrest.conf \
+--stanza=seg%c archive-push %p'" --skipvalidation
+
+gpstop -ar
+
+echo "pgbackrest health check..." > /dev/null
+for i in -1 0 1 2
+do 
+    PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest \
+    --config $PGBACKREST_TEST_DIR/pgbackrest.conf --stanza=seg$i check
+done
+
+# The test scenario starts here
+echo "Creating initial dataset..." > /dev/null
+createdb gpdb_pitr_database
+psql -d gpdb_pitr_database -c \
+"CREATE TABLE t1 (id int, text varchar(255)) DISTRIBUTED BY (id);"
+
+psql -d gpdb_pitr_database -c \
+"INSERT INTO t1 SELECT i, 'text'||i FROM generate_series(1,30) i;"
+
+echo "Creating full backup on master and seg0..." > /dev/null
+for i in -1 0
+do 
+    PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest \
+    --config $PGBACKREST_TEST_DIR/pgbackrest.conf --stanza=seg$i \
+    --type=full backup
+done
+
+echo "Checking the presence of first backup..." > /dev/null
+function check_backup(){
+    segment_backup_dir=$PGBACKREST_TEST_DIR/$TEST_NAME/backup/seg$1
+    current_date=$(date +%Y%m%d)
+    if [[ $(find "$segment_backup_dir" -maxdepth 1 -type d -name \
+        "${current_date}-??????F" -not -empty ) ]];
+    then
+        echo "Found a backup directory for segment $1" > /dev/null
+    else
+        echo "The backup directory for segment $1 was not found" > /dev/null
+        exit 1
+    fi
+}
+for i in -1 0
+do 
+   check_backup $i
+done
+
+echo "Creating additional table..." > /dev/null
+psql -d gpdb_pitr_database -c \
+"CREATE TABLE t2 (id int, text varchar(255)) DISTRIBUTED BY (id);"
+
+psql -d gpdb_pitr_database -c \
+"INSERT INTO t2 SELECT i, 'text'||i FROM generate_series(1,30) i;"
+
+psql -d gpdb_pitr_database -c "SELECT * FROM t2 ORDER BY id;" \
+-o "$PGBACKREST_TEST_DIR/$TEST_NAME/t2_rows_original.out"
+
+echo "Filling the first table with more data at seg0 after the first backup..."\
+> /dev/null 
```

Comment:
I think it is not a problem, when lines length in our code sometimes will be more then 80 symbols, but the code will be more compact